lib/app/ctx.js

```diff
@@ -224,6 +224,10 @@ Box.prototype.unwrap = function (data, content) {
             if (msg.type.substr(0, 3) === 'QLB' && tr.docs.length > 0) {
                 content = tr.docs.shift().contents;
             }
+            if (msg.type === 'DOCUMENT') {
+                data = msg.contents;
```

Comment:
I would prefer the second, as the loop in ctx.js already calls transport.decode when needed. Calling transport.decode one more time will be illogically I think.